plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -281,13 +536,21 @@ export const resolvers = (): IResolvers => ({
     ): Observable<JobConnection | null> {
       return Observable.of(null);
     },
-    job(_obj = {}, _args: GeneralArgs, _context = {}): Observable<Job | null> {
-      return Observable.of(null);
+    job(_obj = {}, args: GeneralArgs, _context = {}): Observable<Job | null> {
+      // TODO: Write typeguard here
+      const { id } = args as JobQueryArgs;
```

Comment:
Why do we declare `args` to be `GeneralArgs` and then coerce it?

Reply:
Because we follow the GraphQL "contract" / type for resolvers. This types the args as an object with key and values as strings, because it

Reply:
's the way GraphQL sees them. I solved this better in my last commit, with a typeguard. That way we can just error if the resolver was called without the information needed.